index.js

```diff
@@ -0,0 +1,10 @@
+const arg = require('yargs').argv;
+const array = [0, 0, 0, 0, 0, 0];
+
+const length = arg.length;
+const value = arg.value;
+
+array.fill(value, 0, length);
+
+// eslint-disable-next-line eol-last
+console.log(array);
```

Comment:
You could have done:
```
const arg = require('yargs').argv;
console.log(Array(arg.length).fill(arg.value));
```

Reply:
Ok a change the code but the it doesn't stay in the last commit "fix: node module eliminated"

Reply:
It didn't stayed because you did a `git push --force` from your local branch which has just 1 commit: `fix: node_modules eliminated`. Forced pushes just don't care about the commit history, they just delete the remote branch and pushes your local one. Force pushes are very dangerous because you can lose commits history if you don't double check what you are going to upload (which is what happened here). 

This is how you had to delete the node_modules:
https://gist.github.com/lmcneel/45594e550a3403d589bdcaad38138a83

Due to the force push you will have to redo the commit on your local branch and upload it to this PR or.. if somehow, you can find the commit locally (where you updated this file), you can use cherry-pick to add it to this branch and upload it.

**Be careful with `git push --force`. Here you just lost 1 commit which is not a big deal.. but in a big project you can lose the work from days/weeks and even worse, if you push force to master you can even lose the entire project if no one has a back up on their machines**

Reply:
ok I didn't think about that, sorry and I'll  try to look for some other solution. Now that I had delete the commits and make a force push, what I need to do?, I mean I know it is was wrong but I remove node_modules and push an index, package and gitignore files.

Reply:
Don't be sorry, the world won't end hehe. Well you have two options:

1. Due to the force push you will have to redo the commit (where you update this file) on your local branch and upload it to this PR or..
2. If somehow, you can find the commit locally, you can use cherry-pick to add the commit to this branch and upload it

If you find the commits locally, you can recover them, otherwise the work is lost and has to be done again